ink/goddess/reward_joi

```diff
@@ -374,7 +374,7 @@ Now, take it nice and slow.
     -> Goddess_SayNumberRangeSlow(8, 10, 2) ->
     -> random_comments ->
     
-    - I want you to show me 20 excrutiatingly slow {hasPenis:strokes|circles}.
+    - I want you to show me 20 excrutiating, slow {hasPenis:strokes|circles}.
```

Comment:
The sentence doesn't quite read right.

It should say, "show me 20 excrutiatingly slow" or "excrutiating and slow"

Alternatively, it's a big word and I'm wondering if "show me 20 painfully slow" would be a little better?